docs/usage-as-operator.md

```diff
@@ -109,3 +118,60 @@ spec:
       - version: 2135.6.0
         urn: "CoreOS:CoreOS:Stable:2135.6.0"
 ```
+
+## `Seed` resource
+
+This provider extension does not support any provider configuration for the `Seed`'s `.spec.provider.providerConfig` field.
+However, it supports to manage backup infrastructure, i.e., you can specify a configuration for the `.spec.backup` field.
```

Comment:
```suggestion
However, it supports managing of backup infrastructure, i.e., you can specify a configuration for the `.spec.backup` field.
```
Not sure about that. We might want to tag @wyb1 :)

Reply:
ping